app/src/main/res/values-de/strings.xml

```diff
@@ -325,11 +325,11 @@
     <string name="remove_the_last_downloaded_version">Die zuletzt heruntergeladene Version entfernen</string>
     <string name="reset_announcements">Ankündigungen zurücksetzen</string>
     <string name="reset_hidden_announcements">Versteckte Ankündigungen zurücksetzen</string>
-    <string name="restore_automatic_backup">Automatische Sicherung wiederherstellen</string>
-    <string name="restore_a_previous_automatic_backup">Eine vorherige automatische Sicherung wiederherstellen</string>
+    <string name="restore_automatic_backup">Automatisch Backup wiederherstellen</string>
+    <string name="restore_a_previous_automatic_backup">Ein vorheriges automatisches Backup wiederherstellen</string>
```

Comment:
Well I don't think I need to spam the previous replies every time there is the word backup. 
But you know..